### PR TITLE
SKR PRO DIN Mount

### PR DESCRIPTION
**Pull request by 0lympu5**
* [ x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ ] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [ x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [ x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [ x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x ] Submitted STL files are oriented and scaled properly for printing.
  * [ x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Mount for SKR PRO boards
#### How was it tested? 
Tested and Fitted to my machine
#### Any background context you want to provide?
NO
#### Screenshots (if appropriate)
One within the readme.md file
#### Further notes
No

### CODE REVIEW

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

- Please create a subdirectory for your mod (e.g. `printer_mods/MrBean66/SKR_PRO_Mount/` and move your files in there
- Please edit `printer_mods/README.md` to add a link to your mod in alphanumeric order
- Please remove spaces from your filenames

File: printer_mods/MrBean66/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+**Printed Parts:**
+- 1x SKR PRO Mount
+- 2x (either) Strected din clamps / Standard voron din clamps
+
+**Hardware:**
+- 4x M3 Heat inserts
+- 4x M3 x 6 BHCS
+- 4x M2 self tapers 
+
+![Image of mod](https://github.com/0lympu5/VoronUsers/blob/master/printer_mods/MrBean66/SKR%20PRO%20Example.png)
```
[the remark this answers is not in the document]
Comment (0lympu5) [reply printer_mods/MrBean66/README.md:14]: Hi there, thanks for the guidance and ill change appropriately. Will try and add a separate file for this mod however it refused to upload when I created the proper folder tree.

I'm confused as to what you mean by alphanumeric link to the mod

**Comment by KiloQubit** [comment]
Let me see if I can fix the directories for you.

**Comment by KiloQubit** [comment]
Ok, I fixed your directory structure, removed spaces from filenames, and removed the empty file.  Please update your README image with a relative link like I mentioned above, and update `printer_mods/README.md` to add a link to your mod.  See the table in that file - find the place in the table where your username would be in alphabetical order (alphanumeric is alphabetical + numbers), and insert a row there - basically copy an existing row and modify it with your name, mod name, like and compatibility checkboxes.  Looks like your mod should go in the table after `Mosher`.

**Comment by 0lympu5** [comment]
Thank you, any issues and ill have another look

**Comment by 0lympu5** [comment]
Okay, I believe all the requested changes have been made.  Hope everything should be set

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**